index.html

```diff
@@ -1,149 +1,166 @@
-         <img class="pic" src="./assets/images/KK.jpg"  alt="KK" />
-       </li>
-       <li id="Nic">
-         <div class="name">Nic</div>
-         <div class="office">Science 1315</div>
-         <img class="pic" src="./assets/images/Nic.jpg"  alt="Nic" />
-       </li>
-       <li id="Peter">
-         <div class="name">Peter</div>
-         <div class="office">Science 1440</div>
-         <img class="pic" src="./assets/images/Peter.jpg"  alt="Peter" />
-       </li>
+<body>
+  <h1 id="main-title">UMM Computer Science</h1>
+  <div class="pslo-courses-container">
+    <div class="section" id="pslo">
+      <h2>Program Student Learning Outcomes for UMM Computer Science</h2>
+      <ol>
+        <li>Students will be able to apply fundamental principles of computer science to solve problems in
+          all core areas of computer science.
+        </li>
+        <li>Students will demonstrate technological flexibility through the ability to employ new sets of
+          tools effectively.
+        </li>
+        <li>Students will be able to communicate technical ideas effectively both orally and in written
+          form.
+        </li>
+        <li>Students will demonstrate their ability to work in groups as part of an effective team.</li>
+        <li>Students will be able to identify and analyze ethical implications involving technology.</li>
+      </ol>
+    </div>
+    <div class="section" id="courses">
+      <h2>Required Courses</h2>
+      <ul>
+        <li class="major-element level-1xxx" id="introductory-sequence">An introductory course sequence
+          <ul>
+            <li><span class="discipline csci">CSci</span> <span class="course-number">1302</span></li>
+            <li>One of:
+              <ul>
+                <li><span class="discipline csci">CSci</span> <span class="course-number">1201</span></li>
+                <li><span class="discipline csci">CSci</span> <span class="course-number">1301</span></li>
+              </ul>
+            </li>
+          </ul>
+        </li>
+        <li class="major-element level-2xxx" id="data-structures">
+          <span class="discipline csci">CSci</span> <span class="course-number">2101</span>
+        </li>
+        <li class="major-element level-3xxx" id="core-courses">Core Courses
+          <ul>
+            <li><span class="discipline csci">CSci</span> <span class="course-number">3402</span></li>
+            <li><span class="discipline csci">CSci</span> <span class="course-number">3403</span></li>
+            <li><span class="discipline csci">CSci</span> <span class="course-number">3501</span></li>
+            <li><span class="discipline csci">CSci</span> <span class="course-number">3601</span></li>
+          </ul>
+        </li>
+        <li class="major-element level-4xxx" id="electives">Ten credits of electives 4xxx so that the split of credits
+          is
+          at least 4 in two of the areas and at least 2 in the third area.
```

Comment:
```suggestion
        <li class="major-element level-4xxx" id="electives">Ten credits of electives 4xxx so that the split of credits
          is at least 4 in two of the areas and at least 2 in the third area.
```